source/Calamari.Tests/Fixtures/ConfigurationVariables/ConfigurationVariablesFixture.cs

```
@@ -88,15 +96,30 @@ public void ReplacesConnectionStrings()
             Assert.AreEqual("Server=bar&bar=123", contents.XPathSelectElement("//connectionStrings/add[@name='MyDb2']").Attribute("connectionString").Value);
         }
 
+        [Test]
+        [ExpectedException(typeof (System.Xml.XmlException))]
+        public void ShouldThrowExceptionForBadConfig()
+        {
+            var variables = new VariableDictionary();
+            PerformTest(GetFixtureResouce("Samples", "Bad.config"), variables);
+        }
+
+        [Test]
+        public void ShouldSupressExceptionForBadConfig()
+        {
```

Names are confusing, should it throw or suppress exceptions for bad config?